source/Memcached-Tests/MDCacheClientTest.class.st

```diff
@@ -2,34 +2,29 @@
 Unit tests for MDCacheClient
 "
 Class {
-	#name : #MDCacheClientTest,
-	#superclass : #MDClientTest,
-	#category : #'Memcached-Tests'
+	#name : 'MDCacheClientTest',
+	#superclass : 'MDClientTest',
+	#category : 'Memcached-Tests',
+	#package : 'Memcached-Tests'
 }
 
-{ #category : #running }
+{ #category : 'running' }
 MDCacheClientTest >> clientClass [ 
 	^ MDCacheClient
 ]
 
-{ #category : #running }
-MDCacheClientTest >> serverList [
-
-	^ #('127.0.0.1:11211')
-]
-
-{ #category : #running }
+{ #category : 'running' }
 MDCacheClientTest >> testStaleSetGet [
 
-	cache remove: #testStaleSetGet.
-	cache set: #testStaleSetGet value: 'test' expires: 100 milliSeconds.
-	150 milliSeconds asDelay wait 
-	"first guy should get nil, has time to reload cache".
-	self assert: (cache get: #testStaleSetGet) isNil 
+	cache remove: 'testStaleSetGet'.
+	cache set: 'testStaleSetGet' value: 'test' expires: 100 milliSeconds.
+	(Delay forMilliseconds: 150) wait.
```

Comment:
Can't we use wait like in the seconds case some lines below?

Reply:
There was no portable extension for milliseconds and I didn't want to add a new one just for a test case.